components/webgpu/lib.rs

```diff
@@ -460,8 +460,8 @@ impl<'a> WGPU<'a> {
                         };
                         let global = &self.global;
                         let result = gfx_select!(buffer_id => global.buffer_map_async(buffer_id, map_range, operation));
-                        if let Err(ref e) = result {
-                            if let Err(w) = sender.send(Err(format!("{:?}", e))) {
```

Comment:
All errors in this file should be `Some(Err`, because the `None` case is only handled at the constellation level when WebGPU is not available, while here we are in the WebGPU backend.